main.go

```diff
@@ -46,23 +46,23 @@ func main() {
 	if err := app.WaitForBuilds(buildSlugs, func(build bitrise.Build) {
 		var failReason string
 		var buildURL = fmt.Sprintf("(https://app.bitrise.io/build/%s)", build.Slug)
-		switch build.Status {
-		case 0:
+
+		if build.IsRunning() {
 			log.Printf("- %s %s %s", build.TriggeredWorkflow, build.StatusText, buildURL)
-		case 1:
+		} else if build.IsSuccessful() {
 			log.Donef("- %s successful %s)", build.TriggeredWorkflow, buildURL)
-		case 2:
+		} else if build.IsFailed() {
 			log.Errorf("- %s failed", build.TriggeredWorkflow)
 			failReason = "failed"
-		case 3:
+		} else if build.IsAborted() {
 			log.Warnf("- %s aborted", build.TriggeredWorkflow)
 			failReason = "aborted"
-		case 4:
+		} else if build.IsAbortedWithSuccess() {
 			log.Infof("- %s cancelled", build.TriggeredWorkflow)
 			failReason = "cancelled"
```

Comment:
We don't need to set this `failReason` here as it won't go into the abort branch where we would use it.

Reply:
Ah good catch. Updated.